src/Engine/GlideSolvers/GlidePolar.hpp

```diff
@@ -66,7 +66,7 @@ class GlidePolar
 
   /** Clean ratio (1=clean, 0=100% bugs) */
   double bugs;
-  /** Ballast ratio (litres) */
+  /** Ballast (litres alias kg) */
```

Comment:
This change is good, but is unrelated to this commit; actually, we should better never refer to "ballast litres" at all, because the ballast volume is not relevant for anything.

Reply:
It is related. The commit/branch tries to clean up the masses mess and unclear code comments are a target.

Reply:
This commit is documented as "Split dry_mass into empty + crew mass, attached reference mass to polar shape", and this change is confusing - not because it is bad, but it does not "split dry_mass into empty+crew mass".